pytorch_lightning/utilities/imports.py

```diff
@@ -84,6 +84,7 @@ def _compare_version(package: str, op, version) -> bool:
 _NATIVE_AMP_AVAILABLE = _module_available("torch.cuda.amp") and hasattr(torch.cuda.amp, "autocast")
 _OMEGACONF_AVAILABLE = _module_available("omegaconf")
 _POPTORCH_AVAILABLE = _module_available("poptorch")
+_RICH_AVAILABLE = _module_available("rich")
```

Comment:
@kaushikb11 - is there a version of rich we should specify here that's compatible with the progress bar features?